projects/ontimize-web-ngx/src/lib/services/local-storage.service.ts

```diff
@@ -8,12 +8,10 @@ import { SessionInfo } from '../types/session-info.type';
 import { ObservableWrapper } from '../util/async';
 import { Util } from '../util/util';
 import { AuthService } from './auth.service';
+import { ILocalStorageService } from '../interfaces/local-service.interface';
 
-
-@Injectable({
-  providedIn: 'root'
-})
-export class LocalStorageService {
+@Injectable()
```

Comment:
It is not a singleton anymore?

Reply:
No, fhe factory is responsible for creating this service `LocalStorageService` or the one provided in the injection token O_LOCALSTORAGE_SERVICE